utils/constants.ts

```diff
@@ -20,4 +20,8 @@ export const COLORS = {
   green: "133deg 68% 39%",
   white: "0deg 100% 100%",
   black: "0deg 0% 0%",
+  red: {
+    100: "0deg 0% 0%",
```

Comment:
the `100` is black and the other one already exists as `primary`

Reply:
Ok, changed!